components/host-dashboard/reports-section/HostFeesSectionHistorical.js

```diff
@@ -109,11 +109,6 @@ const getChartOptions = (intl, hostCurrency) => ({
       formatter: value => formatAmountForLegend(value, hostCurrency, intl.locale),
     },
   },
-  tooltip: {
-    y: {
-      formatter: value => formatAmountForLegend(value, hostCurrency, intl.locale),
-    },
-  },
```

Comment:
@Betree : I've realized that we don't need this duplicated tooltip formatter. [According to the docs](https://apexcharts.com/docs/options/tooltip/);

>To format the Y-axis values of tooltip, you can define a custom formatter function. By default, these values will be formatted according [yaxis.labels.formatter](https://apexcharts.com/docs/options/yaxis/#formatter) function which will be overrided by this function if you define it.

So I've removed those. 🙂

Reply:
The initial purpose of this helper was to have a different value displayed between the y axis and the tooltip: 
- In the y axis, we want compact values to save space
- In the tooltip, we want to make sure we don't round anything and that the full amount is displayed

I'll be restoring it with this intent in https://github.com/opencollective/opencollective-frontend/pull/7715